### ISSUE TITLE
typeof(<bigintvec>) gives "?unknown type?"

### ISSUE THREAD

**Issue by tornaria** [Singular/Singular#1213]
In 4.3.2p10:
```
> ring r = 0,x,dp;
> typeof(hilb(std(ideal(x)),1));
intvec
> type(hilb(std(ideal(x)),1));
// _ intvec 1,-1,0
```

But in 4.3.2p16:
```
> ring r = 0,x,dp;
> typeof(hilb(std(ideal(x)),1));
?unknown type?
> type(hilb(std(ideal(x)),1));
// _ bigintvec 1,-1,0
```

Maybe `case BIGINTVEC_CMD:` is missing [here](https://github.com/Singular/Singular/blob/spielwiese/Singular/iparith.cc#L5448) ?

**Comment by hannes14** [comment]
Sorry, that is part of the upcoming new version of Singular.
Thanks for the hint, but for version 4.3.2 bigintvec support is minimal, there is much more missing

**Comment by tornaria** [comment]
I see, but this is included in 4.3.2p16 and it causes trouble for sagemath (https://github.com/sagemath/sage/pull/37492)

**Comment by hannes14** [comment]
fixed with the release of version 4.4.0
